client/src/main/java/org/spine3/base/Identifiers.java

```diff
@@ -20,55 +20,33 @@
 
 package org.spine3.base;
 
-import com.google.common.base.Function;
-import com.google.common.collect.ImmutableMap;
 import com.google.protobuf.Any;
 import com.google.protobuf.Message;
-import com.google.protobuf.MessageOrBuilder;
 import com.google.protobuf.StringValue;
-import com.google.protobuf.Timestamp;
 import com.google.protobuf.UInt32Value;
 import com.google.protobuf.UInt64Value;
-import com.google.protobuf.util.Timestamps;
 import org.spine3.Internal;
 
 import javax.annotation.Nullable;
-import java.util.Collection;
-import java.util.Map;
 import java.util.UUID;
-import java.util.regex.Pattern;
-
-import static com.google.common.base.Preconditions.checkNotNull;
-import static com.google.common.collect.Maps.newHashMap;
-import static com.google.protobuf.TextFormat.shortDebugString;
-import static java.util.Collections.synchronizedMap;
-import static org.spine3.protobuf.AnyPacker.unpack;
 
 /**
  * This class manages conversion of identifies to/from string.
  *
- * <p>In addition to utility methods for the conversion, it provides {@link ConverterRegistry}
+ * <p>In addition to utility methods for the conversion, it provides {@link StringifierRegistry}
  * which allows to provide custom conversion logic for user-defined types of identifies.
```

Comment:
Well spotted. This text is no longer applicable because the registry is now a separate class.